test/bin/pyutils/build_bootc_images.py

```diff
@@ -239,18 +240,20 @@ def process_containerfile(groupdir, containerfile, dry_run):
                 "-t", cf_outname, "-f", cf_outfile,
                 IMAGEDIR
             ]
+            start = time.time()
             common.retry_on_exception(3, common.run_command_in_shell, build_args, dry_run, logfile, logfile)
-            common.record_junit(cf_path, "build-container", "OK")
+            common.record_junit(cf_path, "build-container", "OK", start)
 
             push_args = [
                 "sudo", "podman", "push",
                 cf_outname,
                 f"{MIRROR_REGISTRY}/{cf_outname}"
             ]
+            start = time.time()
             common.run_command_in_shell(push_args, dry_run, logfile, logfile)
-            common.record_junit(cf_path, "push-container", "OK")
+            common.record_junit(cf_path, "push-container", "OK", start)
     except Exception:
-        common.record_junit(cf_path, "process-container", "FAILED")
+        common.record_junit(cf_path, "process-container", "FAILED", log_filepath=cf_logfile)
```

Comment:
Let's pass the start here as well? It might sometimes be interesting how long an operation takes until it fails.